docs/docs/ab_testing.md

```diff
@@ -0,0 +1,106 @@
+Pathfora supports A/B testing on a global level as well as on a targeted audience. You can split a pool of users to show some proportion of them a group of modules "A" and the other portion a group "B".
+
+
+``` javascript
+var moduleTest = pathfora.ABTest({
+  id: 'ab-test-id',
+  type: '50/50',
+  groups: [
+    [ moduleA ],
+    [ ]
+  ]
+});
+
+pathfora.initializeABTesting([ moduleTest ]);
+pathfora.initializeWidgets([ moduleA ]);
+```
+
+## ABTest
+
+Pathfora has a special configuration method for A/B Testing. This configuration is then used as a parameter for [initializeABTesting](/api/methods/#initializeabtesting).
+
+<table>
+  <thead>
+    <tr>
+      <td colspan="3" align="center"><code>ABTest</code> settings object</td>
+    </tr>
+    <tr>
+      <th>Key</th>
+      <th>Type</th>
+      <th>Behavior</th>
+    </tr>
+  </thead>
+
+  <tr>
+    <td>id</td>
+    <td>string</td>
+    <td>unique identifier for the AB Test</td>
+  </tr>
+  <tr>
+    <td>type</td>
+    <td>string</td>
+    <td>the proportional split between A/B (currently only <code>50/50</code> is supported)</td>
+  </tr>
+  <tr>
+    <td>groups</td>
+    <td>array</td>
+    <td>jagged array defining the A/B groups that each module belongs to</td>
+  </tr>
+</table>
+
+
+## Global
+For a global A/B test, some proportion of all users will see module "A" while the remaining will see module "B". 
+
+In the example below roughly half of all users will be shown a gated form (A) and the other half will see a form modal (B).
+
+### All Users Form/SiteGate Test - [Live Preview](examples/preview/ab-testing/global.html)
```

Comment:
for the live previews should we just use normal links to we can open in a new window rather than load in the current window?